tests/microbenchmarks/bench_moe/routing/native_logits.py

```
@@ -0,0 +1,296 @@
+    materialized_ids: torch.Tensor,
+    materialized_scales: torch.Tensor,
+):
+    """Return a ``routing_method.apply`` wrapper returning the plan directly."""
+
+    def supplied_apply(router_logits):
+        # Honour the original apply to preserve dtypes/device when possible,
+        # but discard its result.
+        try:
+            _ = apply_orig(router_logits)
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)